linux/kernel/updating.md

```diff
@@ -1,32 +1,9 @@
 # Updating the kernel
 
-If you use the standard Raspbian update/upgrade process (found [here](../../raspbian/updating.md)), this will automatically update the kernel to the latest stable version. This is the recommended procedure. However, in certain circumstances, you may wish to update to the latest 'bleeding edge' or test kernel.
+If you use the standard Raspbian update/upgrade process (found [here](../../raspbian/updating.md)), this will automatically update the kernel to the latest stable version. This is the recommended procedure. However, in certain circumstances, you may wish to update to the latest 'bleeding edge' or test kernel. You should only do this if recommended to do so by a Raspberry Pi engineer, or there is a specific feature only available in this latest software.
```

Comment:
House style would probably prefer the implicit "if" to be explicit ("or if there is").